src/site/content/en/blog/smoothness/index.md

```diff
@@ -0,0 +1,560 @@
+3. *Percentage Dropped* over a given time period.
+
+What matters is: _the proportion of time waiting for important
+updates_. We think this matches the natural way users experience the smoothness
+of web content in practice. So far, we have been using the following as an
+initial set of metrics:
+
+- **Average Percent Dropped:** for all non-idle animation frames throughout the
+  whole timeline
+- **Worst Case of Percent Dropped Frames:** as measured over 1 second sliding
+  windows of time.
+- **95th percentile of Percent Dropped Frames:** as measured over 1 second
+  sliding windows of time.
+
+You can find these scores in some Chrome developer tools today. While these
+metrics focus only on overall frame throughput, we are also evaluating other
+factors, such as frame latency.
+
+{% Aside %}
+  These names are somewhat historical. The "Percent Dropped" naming no longer
+  matches any literal "was a frame dropped" concept. As the previous section
+  covered, each animation frame is a complicated arrangement of conditions
+  leading to an overall probability. Expect these names to evolve over time.
+{% endAside %}
+
+## Try it yourself in developer tooling!
+
+### Performance HUD
+
+Chromium has a neat Performance HUD hidden behind a flag
+(`chrome://flags/#show-performance-metrics-hud`). In it, you can find live
+scores for things like Core Web Vitals and also a few experimental definitions
+for animation smoothness based on *Percent Dropped Frames* over time.
+
+{% Img src="image/eqprBhZUGfb8WYnumQ9ljAxRrA72/hblBG5oYxWsUOua0tW3C.png",
+alt="Performance HUD", width="800", height="441" %}
+
+### Frame Rendering Stats
+
+[Enable "Frame Rendering
+Stats"](https://developer.chrome.com/docs/devtools/evaluate-performance/reference/#fps-meter)
+in DevTools via Rendering settings to see a live view of new animation frames,
+which are color coded to differentiate partial updates from fully-dropped frame
+updates. The reported fps is for fully-presented frames only.
+
+{% Img src="image/eqprBhZUGfb8WYnumQ9ljAxRrA72/xazCKFNCWwjviBGS9JO6.png",
+alt="Frame rendering stats", width="398", height="372" %}
+
+### Frame Viewer in DevTools performance profile recordings
+
+The [DevTools performance
+panel](https://developer.chrome.com/docs/devtools/evaluate-performance/) has
+long had a [Frames
+viewer](https://developer.chrome.com/docs/devtools/evaluate-performance/reference/#frames).
+However, it had grown a bit out of sync with how the modern rendering pipeline
+actually works. There has been plenty of recent improvement, even in the latest
+Chrome Canary, which we think will greatly ease debugging animation issues.
+
+Today you will find that frames in the frames viewer are better aligned with
+vsync boundaries, and color coded based on status. There is still not full
+visualization for all the nuances outlined above, but we're planning to add more
+in the near future.
+
+{% Img src="image/eqprBhZUGfb8WYnumQ9ljAxRrA72/xZTBWc5Ox2cKhSSaK9rS.png",
```

Comment:
As discussed, I uploaded two alternative screenshots.  One for the scrolling test page above, another for Youtube.

{% Img src="image/nM10iAStORWg0YlDbPc2czmpRRr2/ou89sgoFeg2481KVUWmF.png", alt="ALT_TEXT_HERE", width="800", height="148" %}
{% Img src="image/nM10iAStORWg0YlDbPc2czmpRRr2/JHr1tEbP3eBmVHYY6JEl.png", alt="ALT_TEXT_HERE", width="800", height="148" %}

Reply:
Thanks, I'll use the first one as I think it better shows the "partially presented frame" concept.